src/components/Settings/People/People.messages.js

```diff
@@ -54,6 +54,10 @@ export default defineMessages({
     defaultMessage:
       'Once you delete an account, there is no going back. Please be certain.'
   },
+  deleteAccountFinal: {
+    id: 'cboard.components.Settings.People.deleteAccountFinal',
+    defaultMessage: 'To confirm, type "delete-account" in the box below.'
```

Comment:
Chage the text to: "To confirm, type "delete-account"."

Reply:
done